libraries/utilities/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ set(sources
    tempdir.cpp
    words.cpp
    logging_config.cpp
+   SysLogger.cpp
```

Comment:
I think we should use google style guide, so instead of CamelCase https://google.github.io/styleguide/cppguide.html#File_Names underscores